src/Class/WinningNumbers.js

```diff
@@ -0,0 +1,109 @@
+import {
+  LOTTO_COUNT,
+  LOTTO_MAX_NUMBER,
+  LOTTO_MIN_NUMBER,
+  MATCH_FIVE_BONUS,
+  MATCH_FIVE,
+  RESULTS_ORDER,
+  MATCH_THREE,
+} from "../constants/lotto.js";
+
+class WinningNumbers {
+  #winningNumber;
+  #bonusNumber;
```

Comment:
지금 객체로 만들어서 쓰고 있는데요,
문제는 객체의 장점을 제대로 누리지 못하고 있어요!

현재의 모습은 "비슷한 용도의 메소드를 클래스로 묶어주자" 라는 느낌입니다.

객체로 정의해서 사용할 때의 장점 중 하나가 "인스턴스로 존재한다는 것은 모든 검증이 끝났다는 것" 이라고 생각하는데요

가령, winningNumber와 bonusNumber를 생성자로 받아들일 때 지금은 아무 조치를 취하지 않고 있어요.
아마 밖에서 검증해서 만들어주는거겠죠?

그렇다면 WinningNumbers 클래스를 사용하는 입장에서 신뢰할 수 있을까요?

지금은 없지만, `new Lotto()` 를 통해 만들어낸 lotto instance는 lotto의 모든 조건을 만족한 상태이고, 
다른 곳에서 쓰일 때에도 `lotto instanceof Lotto` 이렇게 lotto의 instance인지 검증하면 모든 로또로서의 조건을 만족한다고 볼 수 있는거죠.

지금은 WinningNumbers는 객체보단 함수의 덩어리 모음 처럼 느껴져요!
```suggestion
class WinningNumbers {
  #winningNumber;
  #bonusNumber;
```